.openshift-ci_cd/Jenkinsfile

```diff
@@ -23,4 +29,13 @@ node ('master') {
     sh 'oc delete route nodejs-ex-dev || true'  // Bugfix for https://github.com/openshift/origin/pull/9425 not getting into latest OpenShift release
     sh "oc process -f .openshift-ci_cd/deployment-template.yaml -v 'APP_NAME=nodejs-ex-dev,IMAGE_STREAM_NAMESPACE=${k8s_namespace},IMAGE_STREAM_TAG=nodejs-ex:latest' | oc apply -f -"
     sh 'oc annotate is nodejs-ex misc_$(date +%s)=foo || true'  // Hack for 0-2 minute pause caused by https://github.com/openshift/origin/issues/9018
+
+    stage 'Run Integration Tests'
+    sh 'npm config set prefix=$HOME/node && export PATH=$HOME/node/bin:$PATH && npm install -r package.json'
+    //sh 'ROUTE=`oc get routes | grep nodejs-ex-dev | awk \'{print $2}\'` && IP=$ROUTE node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit | tee report.xml'
+    sh "IP=nodejs-ex-dev.${k8s_namespace}.svc.cluster.local node_modules/mocha/bin/mocha --timeout 5000 tests/app_test.js --reporter xunit | tee report.xml"
```

Comment:
I have tried with IP=nodejs-ex-dev, it was failing hence thought of trying this out, both of them return the same IP and it is failing still.